checks/tasks/__init__.py

```diff
@@ -1,7 +1,9 @@
 # Copyright: 2019, NLnet Labs and the Internet.nl contributors
 # SPDX-License-Identifier: Apache-2.0
 import os
+import requests
```

Comment:
My impression was that `http_get` (built upon `http.client`) does not support persistent connections, while `requests` does. This may be a case of premature optimization. Thoughts @gthess?

Reply:
I just noticed that the required bits were actually missing. They are now included as of 4a028a3.

Reply:
Just the premature optimization and the added requirement :) No objection though.